### PR TITLE
rename disableRenderer to throttleRenderer

### PR DESCRIPTION
**Pull request by albert--**

### CODE REVIEW

File: autoPlay.user.js [hunk 1]
```diff
@@ -636,10 +636,10 @@ function toggleAutoRefresh(event) {
 }
 
 function toggleRenderer(event) {
```
Comment (chauffer) [inline autoPlay.user.js]: While you're at it, change the name of this function as well.

File: autoPlay.user.js [hunk 2]
```diff
@@ -264,7 +264,7 @@ function firstRun() {
 	options1.appendChild(makeCheckBox("removeFlinching", "Remove flinching effects", removeFlinching, handleEvent, true));
 	options1.appendChild(makeCheckBox("removeCritText", "Remove crit text", removeCritText, toggleCritText, false));
 	options1.appendChild(makeCheckBox("removeAllText", "Remove all text", removeAllText, toggleAllText, false));
-	options1.appendChild(makeCheckBox("disableRenderer", "Throttle game renderer", disableRenderer, toggleRenderer, false));
+	options1.appendChild(makeCheckBox("throttleRenderer", "Throttle game renderer", throttleRenderer, toggleRenderer, false));
```
Comment (iakat) [inline autoPlay.user.js]: And here.

File: autoPlay.user.js (hunk 1 above)
Comment (iakat) [inline autoPlay.user.js]: While you're at it, change the name of this function as well.

**Comment by albert--** [comment]
What should I change it to? throttleGameRenderer?

**Comment by iakat** [comment]
Yeah, that works.

**Comment by xPaw** [comment]
Shouldn't it begin with `toggle` to match others?

**Comment by iakat** [comment]
In that case, `toggleGameRendererThrottle`.

**Comment by albert--** [comment]
Should I also change throttleRenderer to throttleGameRenderer?

**Comment by iakat** [comment]
Yes, also make sure to squash the commits.

**Pull request closed without merging**